solutions/src/main/java/ru/sberbank/school/task06/CountMapImpl.java

```diff
@@ -0,0 +1,47 @@
+package ru.sberbank.school.task06;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CountMapImpl implements CountMap<String> {
+
+    private Map<String, Integer> map = new HashMap<>();
+
+    @Override
+    public void add(String key) {
+       map.put(key, map.getOrDefault(key, 0));
```

Comment:
&#39;method def&#39; child has incorrect indentation level 7, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1263319725>*